src/app/components/auth-modal/auth-modal.component.html

```diff
@@ -1 +1,20 @@
-<p>auth-modal works!</p>
+<mat-dialog-content class="auth_wrapper">
+    <header>
+      <mat-button-toggle-group class="header_buttons" name="autharization">
+        <mat-button-toggle class="header__button" checked="true" value="logIn" #login
+          >Log in</mat-button-toggle
+        >
+        <mat-button-toggle class="header__button" value="signUp" #signUp>Sign up</mat-button-toggle>
+      </mat-button-toggle-group>
+    </header>
+    <div class="OAuth_wrapper">
+      <button mat-raised-button class="header__button OAuth__button google" id="googleButton">
+        Continue with Google
+      </button>
+      <button mat-raised-button class="header__button OAuth__button fb" id="googleButton">
+        Continue with Facebook
+      </button>
```

Comment:
Is these buttons are part of `register` component? If so, then please, move them to appropriate place in `register` component

Reply:
Is social buttons are common for login and register by design? If so, then never mind

Reply:
yes

![image](https://user-images.githubusercontent.com/62570834/236027569-da5ab541-9ccb-4967-ba15-99db9bbefb13.png)